### PR TITLE
#168262997 fixing dashboard (solving feedback)

### PR DESCRIPTION
**Pull request by Jajabenit250**
#### What does this PR do?
Fixing Dashboard  

#### Description of Task to be completed?
- Combining Tables to form one table

#### How should this be manually tested?
Clone the repository and cd into it. In UI folder open the file in a browser.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#168262997

#### Screenshots (if appropriate)
FireShot Capture 021 - Admin Dashboard -

Questions:

### CODE REVIEW

File: UI/css/main.css [hunk 1]
```diff
@@ -1267,93 +1122,5 @@ a{
...
-}
- .fmentors-border-pale-yellow,.fmentors-hover-border-pale-yellow:hover{
-    border-color:#ffffcc!important
-}
-.fmentors-border-pale-blue,.fmentors-hover-border-pale-blue:hover{
-    border-color:#e7ffff!important
-}
\ No newline at end of file
+ 
+
```
Comment (oesukam) [inline UI/css/main.css:1138]: Add a new line at the end of the file

**Review by oesukam — CHANGES_REQUESTED** [review]
You should reduce the size the of the `Accept`z or `Reject` button

File: UI/css/main.css (hunk 1 above)
Comment (Jajabenit250) [reply UI/css/main.css]: changes applied

File: UI/css/main.css [hunk 2]
```diff
@@ -1122,5 +1122,4 @@ a{
  .fmentors-text-dark-grey,.fmentors-hover-text-dark-grey:hover,.fmentors-text-dark-gray,.fmentors-hover-text-dark-gray:hover{
     color:#3a3a3a!important
 }
-
```
Comment (Jajabenit250) [inline UI/css/main.css]: applied

File: UI/css/main.css (hunk 1 above)
Comment (Jajabenit250) [reply UI/css/main.css:1138]: all changes applied

**Review by oesukam — APPROVED** [review]

**Pull request merged**